frontend/src/components/BillListItem.js

```diff
@@ -81,23 +47,73 @@ export default function BillListItem(props) {
   const completed = (step) => stepCompleted(billData, step);
 
   return (
-    <TableRow key={billData.printNo}>
-      <TableCell component="th" scope="row" colSpan={2} align="center" className={c.billName}>
-        <Link target="_blank" href={billURL}>
-          {fullBillName}
-        </Link>
-        <br />
-        {billData.title}
-      </TableCell>
-      <TableCell className={c.greenBG}></TableCell>
-      <TableCell className={completed('In Committee') ? c.greenBG : c.none}></TableCell>
-      <TableCell className={completed('On Floor Calendar') ? c.greenBG : c.none}></TableCell>
-      <TableCell className={completed('Passed Senate') ? c.greenBG : c.none}></TableCell>
-      <TableCell className={completed('Passed Assembly') ? c.greenBG : c.none}></TableCell>
-      <TableCell className={completed('Delivered to Governor') ? c.greenBG : c.none}></TableCell>
-      <TableCell className={completed('Signed by Governor') ? c.greenBG : c.none} align="center">
-        {completed('Signed by Governor') ? `SIGNED: ${billData.status.actionDate}` : ''}
-      </TableCell>
-    </TableRow>
+    <div style={rowStyle} className={billData.printNo}>
+      <div className="bill-description">
+        <a target="_blank" rel="noopener noreferrer" href={billURL}><h2>{fullBillName}</h2></a>
+        <p>{billData.title}</p>
+      </div>
+      <div className="overall-status passed">
+        <p>
+        {completed("Signed by Governor")
+          ? `SIGNED: ${billData.status.actionDate}`
+          : ""}
+        </p>
+      </div>
+      <div className="introduced">
+        <svg className="icon status__icon">
+          <use xlinkHref={`${Icons}#icon--yes24`} />
+        </svg>
+      </div>
+      <div className="in-committee">
+        {completed("In Committee") && (
+          <svg className="icon status__icon">
+            <use xlinkHref={`${Icons}#icon--yes24`} />
+          </svg>
+        )}
+      </div>
+      <div className="on-floor">
+        {completed("On Floor Calendar") && (
+          <svg className="icon status__icon">
+            <use xlinkHref={`${Icons}#icon--yes24`} />
+          </svg>
+        )}
+      </div>
+      <div className="pass-two">
+        <div className="pass-senate">
```

Comment:
One of the two blank SVGs. You can test by clicking "Passed Assembly" or "Passed Senate."